.github/workflows/ci.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
 
   integration-test:
     strategy:
-      max-parallel: 1
+      # max-parallel: 1
```

Comment:
**suggestion/praise:** Thank you for removing this! If the CI has been behaving, I'm 100% fine with removing `max-parallel` entirely! Saves us a lot of hassle.